doc/cjdns.md

```diff
@@ -23,17 +24,35 @@ somewhat centralized. I2P connections have a source address and I2P is slow.
 CJDNS is fast but does not hide the sender and the recipient from intermediate
 routers.
 
-## Installing CJDNS and connecting to the network
+## Installing CJDNS and finding a friend to connect to the network
 
 To install and set up CJDNS, follow the instructions at
-https://github.com/cjdelisle/cjdns#cjdns.
+https://github.com/cjdelisle/cjdns#how-to-install-cjdns.
 
-Don't skip steps
+You need to initiate an outbound connection to a peer on the CJDNS network
+before it will work with your Bitcoin Core node. This is described in steps
 ["2. Find a friend"](https://github.com/cjdelisle/cjdns#2-find-a-friend) and
 ["3. Connect your node to your friend's
-node"](https://github.com/cjdelisle/cjdns#3-connect-your-node-to-your-friends-node).
-You need to be connected to the CJDNS network before it will work with your
-Bitcoin Core node.
+node"](https://github.com/cjdelisle/cjdns#3-connect-your-node-to-your-friends-node)
+in the CJDNS documentation.
+
+One quick way to accomplish these two steps is to query for available public
+peers on [Hyperboria](https://github.com/hyperboria) by running the following
+
+```
+git clone https://github.com/hyperboria/peers hyperboria-peers
+cd hyperboria-peers
+./testAvailable.py
+```
+
+and to choose a friend from among the peers returned in the
+`hyperboria-peers/<continent/<country>/<city>` directories.
+
+You can then copy/paste its credentials into the relevant IPv4 or IPv6
+"connectTo" JSON object in the `cjdroute.conf` file in your CJDNS directory and
```

Comment:
Maybe this is platform specific, but I'm not sure there always is a "CJDNS directory"? Perhaps you could phrase it as "in the `cjdroute.conf` file you created in the setup phase"?

Reply:
Done